### DISCUSSION TITLE
Default values ?

### DISCUSSION THREAD

**Discussion by itecaxuodel** [emoacht/Monitorian#554] [General]
It may be a really stupid question lol but there it is :

I just installed Monitorian, i added the Contrast changing slide. I moved it but didn't notice what was the default contrast %(value) before moving it lol. Is there a way to know what was each default values (%) for the brightness and the contrast ?

Thanks in advance

**Comment by emoacht** [comment]
The default values depend on the specific monitor. This app just gets the current values from a monitor.
If you really need to know, try factory reset of the monitor.

**Reply by itecaxuodel** [reply]
Oh okokk, i'll try that !

Thanks for your answer mate
